src/components/AnnotationTarget/__tests__/AnnotationTarget-test.tsx

```diff
@@ -15,8 +18,8 @@ describe('AnnotationTarget', () => {
         preventDefault: jest.fn(),
         stopPropagation: jest.fn(),
     };
-    const getWrapper = (props = {}): ShallowWrapper => {
-        return shallow(
+    const getWrapper = (props = {}): ReactWrapper => {
+        return mount(
```

Comment:
instead of a full mount, would a shallow combined with wrapper.find('a').dive() work?

Reply:
Shallow doesn't currently work with useEffect and throws hooks errors.